bin/test_build

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash -e
 
-test=test_bacnet,test_hold,test_mudgee,test_brute,test_discover,usi,test_ipaddr
 setup=aardvark,aardvark2,default,networking,switch
 faux=faux1,faux2
 faucet=faucet,gauge
+test=test_bacnet,test_hold,test_mudgee,test_brute,test_discover,usi,test_ipaddr
```

Comment:
call this something else besides "test" -- "base"?

Reply:
Yup. Changed.